Mathlib/Data/ENat/Lattice.lean

```diff
@@ -3,8 +3,9 @@ Copyright (c) 2022 Yury Kudryashov. All rights reserved.
 Released under Apache 2.0 license as described in the file LICENSE.
 Authors: Yury Kudryashov
 -/
-import Mathlib.Data.Nat.Lattice
+import Mathlib.Algebra.Order.BigOperators.Group.Finset
```

Comment:
I think that it's better to create a file `ENat.BigOperators` instead of adding heavy imports to this one.

Reply:
Actually, the heavy imports can easily be avoided. See #17939

Reply:
The imports are now only increasing by 22 files. Do you still think it's worth creating a separate one?